Classes/Service/FileGenerator.php

```diff
@@ -129,7 +129,7 @@ class FileGenerator
     ];
     /**
      * @var \EBT\ExtensionBuilder\Service\LocalizationService
-     * @inject
+     * @TYPO3\CMS\Extbase\Annotation\Inject
```

Comment:
Isn't this discouraged as well?
We should use constructor injection or explicit inject methods.

Reply:
hmm, I just changed it since it's obvious that we need name spaced annotations. Can't see any deprecation message. So you know more about this?

Reply:
Ok, found it myself. So I will replace the inject annotations - but in another PR

Reply:
> Inject annotations are still necessary in TYPO3 9.x 
? We use injectMethods since v7

Reply:
Yes of course - but the annotation is needed if you don't use inject methods. I want to change the inject stuff in another PR. I should not have updated the inject annotations - just saw this https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Deprecation-82869-ReplaceInjectWithTYPO3CMSExtbaseAnnotationInject.html?highlight=inject and decided to remove the plain inject annotation...